### ISSUE TITLE
Alert Transports page doesn't load after upgrade to 24.1.0-5

### ISSUE THREAD

**Issue by vhuk** [librenms/librenms#15716]
### The problem

Alert Transport page fails to load after upgrading to version 24.1.0-5-g2a7892b4c - Mon Jan 08 2024 20:49:57 GMT+0200. This is caused by error:

Class "LibreNMS\Alert\Transport\Jiraservicemanagement" not found {"userId":39,"exception":"[object] (Error(code: 0): Class \"LibreNMS\\Alert\\Transport\\Jiraservicemanagement\" not found at /opt/librenms/LibreNMS/Alert/Transport.php:35)"} 

### Output of ./validate.php

```txt
===========================================
Component | Version
--------- | -------
LibreNMS  | 24.1.0-5-g2a7892b4c (2024-01-08T20:49:57+02:00)
DB Schema | 2023_12_10_130000_historical_data_to_ports_nac (277)
PHP       | 8.2.7
Python    | 3.11.2
Database  | MariaDB 10.11.4-MariaDB-1~deb12u1
RRDTool   | 1.7.2
SNMP      | 5.9.3
===========================================

[OK]    Composer Version: 2.6.6
[OK]    Dependencies up-to-date.
[OK]    Database connection successful
[OK]    Database Schema is current
[OK]    SQL Server meets minimum requirements
[OK]    lower_case_table_names is enabled
[OK]    MySQL engine is optimal
[OK]    Database and column collations are correct
[OK]    Database schema correct
[OK]    MySQl and PHP time match
[OK]    Active pollers found
[OK]    Dispatcher Service not detected
[OK]    Locks are functional
[OK]    Python poller wrapper is polling
[OK]    Redis is functional
[OK]    rrd_dir is writable
[OK]    rrdtool version ok
```

### What was the last working version of LibreNMS?

24.1.0

### Anything in the logs that might be useful for us?

```txt
Class "LibreNMS\Alert\Transport\Jiraservicemanagement" not found {"userId":39,"exception":"[object] (Error(code: 0): Class \"LibreNMS\\Alert\\Transport\\Jiraservicemanagement\" not found at /opt/librenms/LibreNMS/Alert/Transport.php:35)"}
```

**Comment by vhuk** [comment]
This seems to be due to naming: class name is "JiraServiceManagement" and it is in file "JiraServiceManagement.php".

File that attempts to load the class (LibreNMS/Alert/Transport.php) will run the filename through `strtolower()` on L33 and `ucfirst()` L138.

**Comment by PipoCanaja** [comment]
Hi @vhuk 
Is it fixed on your setup after #15717  ?
(it is not on my dev machine, but I don't know if it is my machine or not :D )

**Comment by vhuk** [comment]
@PipoCanaja, This has been fixed by #15717. Thank you all!

**Comment by dejantep** [comment]
It's all good after running ./daily.sh manually
Thank you
